substra/sdk/client.py

```diff
@@ -350,6 +380,31 @@ def add_traintuple(self, data, timeout=False, exist_ok=False):
         # less data when responding to adds). A second GET request hides the discrepancies.
         return self.get_traintuple(get_asset_key(res))
 
+    def add_aggregate_traintuple(self, data, timeout=False, exist_ok=False):
+        """Create new aggregate traintuple asset.
+
+        `data` is a dict object with the following schema:
+
+```
+        {
+            "algo_key": str,
+            "objective_key": str,
+            "data_manager_key": str,
+            "tag": str,
+            "compute_plan_id": str,
```

Comment:
I think this is not compliant with the specification.
The `data_manager_key` is not needed.
The following fields are missing:
- worker
- in_model_ids
- rank